packages/invoices/src/OutboundShipment/DetailView/modals/hooks/useNextItem.tsx

```diff
@@ -0,0 +1,18 @@
+import { Item } from '@openmsupply-client/common';
+import { toItem } from '@openmsupply-client/system';
+import { useOutboundRows } from '../../../api';
+
+export const useNextItem = (currentItemId?: string): Item | null => {
+  const { items } = useOutboundRows();
+  if (!items || !currentItemId) return null;
+
+  const numberOfItems = items.length;
+  const currentIdx = items.findIndex(({ itemId }) => itemId === currentItemId);
+  const nextItem = items[(currentIdx + 1) % numberOfItems];
+
+  if (currentIdx === -1 || currentIdx === numberOfItems - 1 || !nextItem) {
```

Comment:
I prefer having the 'next' stop on the last item, like this - instead of looping around. while the looping is less annoying to use, I think there's potential that people will click next for more than one full cycle of items without realising! Do think it would be helpful to disable the next button when you hit the end of the list though